src/stream/src/executor/group_top_n.rs

```diff
@@ -544,9 +504,7 @@ mod tests {
             &StreamChunk::from_pretty(
                 "  I I I
                 - 10 1 1
-                -  8 1 3
-                -  8 8 2
-                +  8 1 3",
+                -  8 8 2",
```

Comment:
Why the output of tests are different from previous?

Reply:
Because now the message I pass to the next operator only contains the difference from the last message.

Reply:
Suppose I now use B as the group key and sort in order，offset is 0 and limits is 1. Then I pass two chunks to the operator. 
a  b
1  9
2  9

Reply:
Suppose I now use B as the group key and sort in order，offset is 0 and limits is 3. Then I pass two chunks to the operator. 
the first chunk is
     a  b
+  4  9
+  5  9
+  6  9
the second chunk is
   a  b 
+ 1  9
In the previous implementation, when the second message arrived.  I would delete the data passed before and then scan the state table, fetching the top 3 directly and passing it to the downstream operator. So the message passed to the downstream operator is 
     a  b
-  4  9
-  5  9
-  6  9
+ 1  9
+  4  9
+  5  9

Reply:
Got it!